### ISSUE TITLE
Complete example not working

### ISSUE THREAD

**Issue by nicolaevladescu** [cloudposse/terraform-aws-sso#12]
Found a bug? Maybe our [Slack Community](https://slack.cloudposse.com) can help. 

[![Slack Community](https://slack.cloudposse.com/badge.svg)](https://slack.cloudposse.com)

## Describe the Bug
```
│ Error: Invalid for_each argument
│
│   on .terraform/modules/assignments/modules/account-assignments/main.tf line 30, in resource "aws_ssoadmin_account_assignment" "this":
│   30:   for_each = local.assignment_map
│     ├────────────────
│     │ local.assignment_map will be known only after apply
│
│ The "for_each" value depends on resource attributes that cannot be determined until apply, so Terraform cannot predict how many instances will be created. To work around this, use the -target argument to
│ first apply only the resources that the for_each depends on.
```

## Steps to Reproduce
1. Clone this repository
2. `cd ./examples/complete`
3. Modify `main.tf` to replace boilerplate with actual group names and real AWS account ids
4. `terraform init`
5. `terraform plan`
6. See error

## Environment (please complete the following information):

Anything that will help us triage the bug will help. Here are some ideas:
 - OS: Debian 10 on WSL
 - Terraform version: 0.15.1

**Comment by jgrumboe** [comment]
Got the same with TF 0.14.6.
It's because the example relays on module output https://github.com/cloudposse/terraform-aws-sso/blob/master/examples/complete/main.tf#L33 and the arn of the permissionset is only known after apply.
I switched back to using 0.2.1.
Edit: sorry, just saw that there's already PR #13 
Will try it

**Comment by nitrocode** [comment]
The example in the repo should work now.

Could you confirm? @nicolaevladescu @jgrumboe

**Comment by jgrumboe** [comment]
@nitrocode 
No, example/complete isn't working, because new value `permission_set_name` is missing in `examples/complete/main.tf`.
We missed that in the PR.

**Comment by mcalhoun** [comment]
Fixed in #13 and #20.
